packages/playground/src/dashboard/transfer_view.vue

```diff
@@ -192,12 +192,13 @@ function isSameAddress(value: string) {
     return { message: "Cannot transfer to yourself" };
   }
 }
+const INVALID_ADDRESS = 'Invalid Address';
```

Comment:
please extract this in a util file to be reused elsewhere

Reply:
IMO it is not needed for now, we can have a separate issue for unifying invalid address error and have it in types

this pr is to fix the Selenium tests only